readme.md

```diff
@@ -37,6 +37,7 @@ It does the following:
  - Commit group headers in commit listings
  - The commit title for single commits
  - The header for each changed file in pull requests and commits
+ - Each notification, and repo in the notifications list.
```

Comment:
nit: Remove comma before "and". I feel it adds confusion rather than clarity here.